README.md

```diff
@@ -1,24 +1,11 @@
 # hyper-content-for
 
-Allows you to store HTML for use (and re-use) later.
```

Comment:
What about "yield content to another scope"? Is "yield" a word that only rubyists understand?